### DISCUSSION TITLE
Rowy invitation emails will not be sent!

### DISCUSSION THREAD

**Discussion by simongolda** [AntlerVC/firetable#1366] [Support / Q&A] [answered]
Hello everyone, I just have the problem that I can not add more members to my Rowy. I invite the others, enter their mail and it says the e-mail was sent!
The only problem is that I have not received one for days.

**Comment by harinij** [comment, marked as answer]
Hi @simongolda Thanks for pointing it out, we have resolved this issue, if you try again it should be sent straight to your inbox 🤞
